x-pack/test/functional/apps/dashboard/group3/reporting/download_csv.ts

```diff
@@ -96,8 +103,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await clickActionsMenu('EcommerceData');
         await clickDownloadCsv();
 
-        const csvFile = await getDownload(getCsvPath('Ecommerce Data'));
-        expectSnapshot(csvFile).toMatch();
+        const { size, lines } = await getDownloadedFileStats(getCsvPath('Ecommerce Data'));
+        expect({ size, lines }).to.eql({ size: 76137, lines: 457 });
```

Comment:
Initially I tried using `expectSnapshot().toMatchInline()` but I was getting an error when using that form
```
ERROR Error: ENOENT: no such file or directory, open 'download_csv.ts'
          at Object.openSync (node:fs:603:3)
          at readFileSync (node:fs:471:35)
          at saveSnapshotsForFile (/Users/tim/elastic/kibana/node_modules/jest-snapshot/build/InlineSnapshots.js:116:22)
          at saveInlineSnapshots (/Users/tim/elastic/kibana/node_modules/jest-snapshot/build/InlineSnapshots.js:105:5)
          at SnapshotState.fn [as save] (/Users/tim/elastic/kibana/node_modules/jest-snapshot/build/State.js:154:50)
```